samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -898,6 +878,12 @@ class SamzaContainer(
     })
   }
 
+  def startTableManager: Unit = {
+    info("Starting task instance table manager.")
```

Comment:
Fixed

Reply:
nit: String.format("Starting table manager in task instance %s", this.taskName); reads better?

Reply:
However this may not be consistent with current logs. Some examples
    info("Starting task instance stores.")
    info("Starting task instance table manager.")
    info("Registering task instances with metrics.")
    info("Registering task instances with producers.")
    info("Registering task instances with consumers.")